client/src/components/ChatMessage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Message } from '../types';
```

Comment:
Any particular reason we're importing this?

Reply:
I thought to import Message from the types file because it has the props "id" and "text". I assumed that was what you did with your other components.

Reply:
Ah, Message only needs to be imported if you're using a Message data type.